engine/apps/slack/scenarios/paging.py

```diff
@@ -305,19 +305,20 @@ def process_scenario(
         if not user_is_oncall(selected_user):
             # display additional confirmation modal
             metadata = json.loads(payload["view"]["private_metadata"])
-            private_metadata = make_private_metadata(
-                {
-                    "state": payload["view"]["state"],
-                    "input_id_prefix": metadata["input_id_prefix"],
-                    "channel_id": metadata["channel_id"],
-                    "submit_routing_uid": metadata["submit_routing_uid"],
-                    DataKey.USERS: metadata[DataKey.USERS],
-                },
-                selected_user.organization,
-            )
+            private_metadata = {
```

Comment:
Will it work without make_private_metadata? I mean chatops-proxy still need to route payloads sent by modal interactions

Reply:
it's still being called on this line: https://github.com/grafana/oncall/pull/4723/files#diff-ba09b07d53839aef9f2a7f3b6cf2a4a8c50e62e53f3779ed92c401961f941a81R321